src/main/java/seedu/dictionote/logic/commands/EditNoteCommand.java

```diff
@@ -78,7 +78,7 @@ public CommandResult execute(Model model) throws CommandException {
         Note editedNote = createEditedNote(noteToEdit, editNoteDescriptor);
 
         if (noteToEdit.isSameNote(editedNote)) {
-            throw new CommandException(MESSAGE_NOTHING_CHANGE_NOTE);
+            throw new CommandException(MESSAGE_NOT_EDITED);
         }
```

Comment:
can you try edit an existing note with the same content and see the result? It might introduce another bug.